bazel/repositories.bzl

```diff
@@ -167,9 +167,9 @@ def data_dependency():
     http_archive(
         name = "seastar",
         build_file = "//bazel/thirdparty:seastar.BUILD",
-        sha256 = "31b526bb8ddb445d624997bfd2f4f62919a9ddbfeb97ddbff71ea68a291ad211",
-        strip_prefix = "seastar-7820b867bf659a1908e71e06f9b4d6bcea019c70",
-        url = "https://github.com/redpanda-data/seastar/archive/7820b867bf659a1908e71e06f9b4d6bcea019c70.tar.gz",
+        sha256 = "e436b109eb5883bac15984b6dfffcc8dd3c5ad64956f3b34ed9a57e55e39d221",
```

Comment:
you need to check in the updated lock file as well.

Reply:
Nice catch, thanks